js/overrides.js

```diff
@@ -1,61 +1,45 @@
-  // fails due to wrong screenshot size on other sdks
-  'should send region by selector in padded page': {skipEmit: true}
+    /// Updating baselines to JS
+    // Window
+    'check window two times with vg classic': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check window with vg classic': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check window with vg': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check window fully and frame in frame fully with css stitching': {config: {branchName: 'universal-sdk'}}, // was v2
+    'check window fully and frame in frame fully with scroll stitching': {config: {branchName: 'universal-sdk'}}, // was v2
+    'check window fully and frame in frame fully with vg': {config: {branchName: 'universal-sdk'}}, // was v2
+    'check window fully with custom scroll root with css stitching': {config: {branchName: 'universal-sdk'}}, // was v2
+    // Region
+    'check region by coordinates in frame with css stitching': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check region by coordinates in frame with scroll stitching': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check overflowed region by coordinates with css stitching': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check overflowed region by coordinates with scroll stitching': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check regions by coordinates in overflowed frame with scroll stitching': {config: {branchName: 'universal-sdk'}}, // was next
+    'check region by selector with vg classic': {config: {branchName: 'universal-sdk'}}, // was no-fully-by-default
+    // Frame
+    'check frame in frame fully with css stitching': {config: {branchName: 'universal-sdk'}}, // was v1
+    'check frame in frame fully with scroll stitching': {config: {branchName: 'universal-sdk'}}, // was v1
+    'check frame in frame fully with vg': {config: {branchName: 'universal-sdk'}}, // was v1
+    'check frame with vg': {config: {branchName: 'universal-sdk'}}, // was v2 and skipped
+    'check frame fully with css stitching': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check frame with css stitching': {config: {branchName: 'universal-sdk'}}, // was v2 and skipped
+    'check frame with scroll stitching': {config: {branchName: 'universal-sdk'}}, // was v2
+    'check frame after manual switch to frame with vg classic': {config: {branchName: 'universal-sdk'}}, // was no-fully-by-default
+    'check frame fully with vg': {config: {branchName: 'universal-sdk'}}, // was skipped
+    // Assertion
+    'should send floating region by selector with vg': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'should send ignore region by selector with vg': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'should send floating region by coordinates with vg': {config: {branchName: 'universal-sdk'}}, // was skipped
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
    "should send floating region by coordinates with vg": {config: {branchName: 'universal-sdk'}}, // was skipped
```